packages/cpmd/package.py

```diff
@@ -81,6 +104,24 @@ def edit(self, spec, prefix):
             cp.filter('-ffree-line-length-none', '')
             cp.filter('-falign-commons', '-Kalign_commons')
 
+        if spec.satisfies('%intel'):
+            cp.filter('CPP=', 'CPP=/lib/cpp -P -traditional')
+            #cp.filter("CPPFLAGS='-D__Linux", "CPPFLAGS='-D__Linux -DLINUX_IFC")
+            cp.filter("-D__HAS_FFT_DEFAULT", "-D__HAS_FFT_FFTW3")
+            cp.filter('-O3 -pc64', '-O2 -m64 -unroll-aggressive')
+            cp.filter("LFLAGS='-static-intel '", "LFLAGS=")
+
+            # The intel configure file assumes intel-mkl
+            # I'm trying to generalize it
+            if not spec.satisfies('^mkl'):
+                cp.filter(
+                'LIBS=.+',
+                "LIBS='{0}'".format(spec['lapack'].libs.ld_flags)
+            )
```

Comment:
This should work for `intel-mkl` also no ?
In which case it is already above other why the test can be combined with the previous one line 79
`if not spec.satisfies('%intel') or not spec.satisfies('^mkl'):`